schemas/network.yml

```diff
@@ -27,6 +27,21 @@
         Host IP address when the source IP address is the proxy.
       example: 192.1.1.2
 
+    - name: session_id
+      type: keyword
+      description: >
+        This is the session ID or connection ID,
```

Comment:
Based on the description I'm wonder if we should put it into a connection object? See other discussions around connection.

I think there is value for `network` info and connection info.